config/src/main/java/org/hyperledger/besu/config/JsonGenesisConfigOptions.java

```diff
@@ -274,6 +275,11 @@ public OptionalLong getPhoenixBlockNumber() {
     return getOptionalLong("phoenixblock");
   }
 
+  @Override
+  public OptionalLong getEcip1099BlockNumber() {
+    return getOptionalLong("ecip1099block");
+  }
+
```

Comment:
I think we also need to add this ecip here in the code  (in the asMap method) https://github.com/hyperledger/besu/blob/1616d36a6607c1ca400e3485d559bbac720c08f5/config/src/main/java/org/hyperledger/besu/config/JsonGenesisConfigOptions.java#L336

Reply:
Added.